eng/common/scripts/Verify-Links.ps1

```diff
@@ -6,18 +6,20 @@ param (
   # switch that will enable devops specific logging for warnings
   [switch] $devOpsLogging = $false,
   # check the links recurisvely based on recursivePattern
-  [switch] $recursive = $true,
+  [switch] $recursive,
```

Comment:
Why did you change the default here?

Reply:
The IDE reminds me switch defaults the bool field to true.
I tested it a bit. It is not default to true. Will revert the change back.